doc/whats_new/v0.20.rst

```diff
@@ -1105,6 +1105,13 @@ Miscellaneous
   safe and could result in a "pop from empty list" error. :issue:`9569`
   by `Andreas Müller`_.
 
+- |API| The default value of ``n_jobs`` is changed from ``1`` to ``None`` in
+  all related functions and classes. ``n_jobs=None`` means ``unset``. It will
```

Comment:
Maybe link to the glossary for additional information?

Reply:
@rth So your suggestion is link to the glossary and do not explain the meaning of ``n_jobs=None`` in what's new? Or simply add a link to the glossary?

Reply:
I was thinking keeping it as it is, and simply add a link : (for additional information see Glossary)